src/helpers/db/dbStats.js

```diff
@@ -0,0 +1,162 @@
+//#region enums
+
+/**
+ * @enum {number} return values for most db helper functions
+ */
+export const dbReturnType = Object.freeze({
+  isNotOk: -1,
+  isOk: 0,
+  isIn: 1,
+  isNotIn: 2,
+});
+
+/**
+ * @enum {string} user's stats attibutes
+ */
+export const statsKeys = Object.freeze({
+  gifs: "gifs",
+  hello: "hello",
+  hungry: "hungry",
+  reactions: "reactions",
+  rolling: "rolling",
+});
+
+//#endregion
+
+//#region use command
+
+/**
+ * Check if the user is in "accepting stats" user list
+ * @param {object} db Database object
+ * @param {string} userId User id to verify
+ * @returns {dbReturnType} isIn if is in database, isNotIn otherwise
+ */
+const isStatsUser = (db, userId) => {
+  const userIds = db.data.stats.map((obj) => obj.userId);
+  if (userIds.includes(userId)) return dbReturnType.isIn;
+  else return dbReturnType.isNotIn;
+};
+
+/**
+ * Add user id to "accepting stats" user list
+ * @param {object} db Database object
+ * @param {string} userId User id to add
+ * @returns {dbReturnType} isOk if is ok, isIn if user is in db
+ */
+const addStatsUser = (db, userId) => {
+  const data = db.data.stats;
+  if (isStatsUser(db, userId) === dbReturnType.isIn) return dbReturnType.isIn;
+  else {
+    const statsInit = Object.values(statsKeys).map((val) => [val, 0]);
```

Comment:
This variable could be global , so it will only be computed once. And I would have defined it like that : 

Object.values(statsKeys).reduce((acc, val) => {...acc, [val] : 0 }, {})  

Like that, you can get rid of Object.fromEntries and use : 
db.data.stats = [
...data, 
{userId, ...statsInit}
] 

Which seems clearer to me